internal/worker_version_sets.go

```diff
@@ -47,6 +51,27 @@ const (
 	VersioningIntentDefault
 )
 
+// TaskReachability specifies which category of tasks may reach a worker on a versioned task queue.
+// Used both in a reachability query and its response.
+type TaskReachability int
+
+const (
+	// TaskReachabilityNotFetched indicates that reachability for this task
+	// was not fetched
+	TaskReachabilityNotFetched TaskReachability = iota - 1
```

Comment:
I'm not sure I'd put this in this enum - since it's not really a reachability condition. If you're gonna have the separate container for unfetched things, then you won't need it anyway.